order/src/lib.rs

```diff
@@ -0,0 +1,214 @@
+//! Contains the order type as described by the specification with serialization as described by the openapi documentation.
+//!
+//! This is in its own crate because we want to share this module between the orderbook and the solver.
+
+use chrono::{offset::Utc, DateTime};
+use primitive_types::H160;
```

Comment:
Should we use the H160 from ethcontracts-rs? I think this would be better, as later on we will heavily rely on eth-contract and converting between  primitive_types::H160 and ethcontract::H160 might become a hussle.

Reply:
It's the same type so it won't be a problem.